### PR TITLE
Enable the Handsontable data sorting plugin

### PR DESCRIPTION
**Pull request by burnash**
This PR updates the Handsontable version to 0.16.1 and enables data sorting plugin.

### CODE REVIEW

File: neurovault/apps/statmaps/static/css/handsontable.full.min.css [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+ */.htContextMenu{display:none;position:absolute;z-index:1060}.htContextMenu .ht_clone_top,.htContextMenu .ht_clone_left,.htContextMenu .ht_clone_corner,.htContextMenu .ht_clone_debug{display:none}.htContextMenu table.htCore{outline:1px solid #bbb}.htContextMenu .wtBorder{visibility:hidden}.htContextMenu table tbody tr td{background:white;border-width:0;padding:4px 6px 0 6px;cursor:pointer;overflow:hidden;white-space:nowrap;text-overflow:ellipsis}.htContextMenu table tbody tr td:first-child{border:0}.htContextMenu table tbody tr td.htDimmed{font-style:normal;color:#323232}.htContextMenu table tbody tr td.current,.htContextMenu table tbody tr td.zeroclipboard-is-hover{background:#e9e9e9}.htContextMenu table tbody tr td.htSeparator{border-top:1px solid #bbb;height:0;padding:0}.htContextMenu table tbody tr td.htDisabled{color:#999}.htContextMenu table tbody tr td.htDisabled:hover{background:white;color:#999;cursor:default}.htContextMenu table tbody tr td div{padding-left:10px}.htContextMenu table tbody tr td div span.selected{margin-top:-2px;position:absolute;left:4px}/*!
+ * Pikaday
+ * Copyright © 2014 David Bushell | BSD & MIT license | http://dbushell.com/
```
Comment (chrisfilo) [inline neurovault/apps/statmaps/static/css/handsontable.full.min.css]: Codacy Issue found: [The property -webkit-box-sizing is compatible with -moz-box-sizing and should be included as well.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/840891242/issues/source?bid=2767031&fileBranchId=2804008#l15)

Comment (chrisfilo) [inline neurovault/apps/statmaps/static/css/handsontable.full.min.css]: Codacy Issue found: [Duplicate property '*position' found.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/840891242/issues/source?bid=2767031&fileBranchId=2804008#l15)

Comment (chrisfilo) [inline neurovault/apps/statmaps/static/css/handsontable.full.min.css]: Codacy Issue found: [margin can't be used with display: inline.](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/840891242/issues/source?bid=2767031&fileBranchId=2804008#l15)

**Comment by chrisfilo** [comment]
I've updated Codacy to ignore handsometables css and js. You can ignore this test fail.

**Comment by ljchang** [comment]
@burnash I think this update might have broken something with handsontable.  The table isn't rendering for me anymore.

**Comment by chrisfilo** [comment]
Hmm. I've merged it and deployed. Looks ok for the collection I've looked
at.

**Comment by ljchang** [comment]
I'm not sure what's going on.  It definitely still works for my smaller collections, but for any of my bigger ones (n>40) the table doesn't render.

**Comment by burnash** [comment]
@ljchang Thanks for reporting it. I can confirm the bug for Single Subject Thermal Pain. It's strange but there are no errors in developer console.

**Comment by burnash** [comment]
@ljchang failing Handsontable is fixed in #316. Thanks @chrisfilo for quick review & deploy.

**Comment by ljchang** [comment]
Wow you guys are fast!  Everything seems to be working perfectly now.  Thanks @burnash and @chrisfilo!

**Pull request merged**